### PR TITLE
Hints + PROGREssbars (son del PSOE) + Favicon + Buttons visibility + filters

### PR DESCRIPTION
**Pull request by ignapas**
## What do these changes do?

* Progress bars are fixed.
* Add a favicon (little icon in the browser tab), now done properly (no JS)
* Managed visibility of Group/Ungroup buttons
* Fixed service browser's filters

## Related issue number

#1281 #1282 

### @odeimaiz I changed some code to reuse the same ``StudyEditor`` between different studies. Going back to the Dashboard and opening another study now uses the same ``StudyEditor`` instance. I did it because we were attaching a lot of event handlers to a lot of widgets and many copies of the callbacks where being executed. But this could break some stuff... please review :)
* Play with selection (in workbench and node tree) and check buttons' visibility

## Screenshot
cast4pedro

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [x] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: services/web/client/source/class/osparc/desktop/ControlsBar.js [hunk 1]
```diff
@@ -141,13 +150,32 @@ qx.Class.define("osparc.desktop.ControlsBar", {
...
+        this.fireEvent(signalName);
       }, this);
       return button;
+    },
+
+    __updateGroupButtonsVisibility: function(msg) {
+      const selectedNodes = msg.getData();
+      let groupBtnVisibility = "excluded";
+      let ungroupBtnVisibility = "excluded";
+      if (selectedNodes.length > 1) {
```
Comment (odeimaiz) [inline services/web/client/source/class/osparc/desktop/ControlsBar.js]: if (selectedNodes.length)

Grouping a single node should be possible

**Comment by ignapas** [comment]
New changes to test:
* Fixed problems with hints
* Applied permissions to editing node and study names from the ``EditLabel``
* Added permissions to (user clearance)
  * Tags
  * Grouping
* Really solved the progressbar issue

File: services/web/client/source/class/osparc/data/Permissions.js [hunk 2]
```diff
@@ -138,14 +138,17 @@ qx.Class.define("osparc.data.Permissions", {
           "preferences.user.update",
           "preferences.token.create",
           "preferences.token.delete",
+          "preferences.tag",
```
Comment (odeimaiz) [inline services/web/client/source/class/osparc/data/Permissions.js]: The convention we are following for the permissions is:
scope.resource.action

Comment (ignapas) [reply services/web/client/source/class/osparc/data/Permissions.js]: does it make sense to leave it to block all actions inside the preferences view? i am not showing the tab at all...

Comment (pcrespov) [reply services/web/client/source/class/osparc/data/Permissions.js]: @ignapas Please sync from time to time the front-end permissions with [those defined in the backend](https://github.com/ITISFoundation/osparc-simcore/blob/857de39ba6438e97eda2d913e15aedd470598772/services/web/server/src/simcore_service_webserver/security_roles.py)

**Pull request merged**